### DISCUSSION TITLE
are macros able to bind multiple paths for one handler?

### DISCUSSION THREAD

**Discussion by godlockin** [actix/actix-web#1932] [Q&A] [answered]
i need to implement health check api to fit for different monitor bots (don't ask me why several monitors exists :( )

for instance:
 bot A will call http://localhost:8000/health, and expect "hello" as success (health) response
 bot B will call http://localhost:8000/hello, and expect "hello" as success response as well

my question is, is it able to bind both routes (/health, and /hello) into one function?

```rust
#[get("/hello")]
#[get("/health")]
async fn health() -> impl Responder {
    HttpResponse::Ok().body("hello")
}
```

or maybe
```rust
#[get(vec!["/health", "/hello"])]
async fn health() -> impl Responder {
    HttpResponse::Ok().body("hello")
}
```

**Comment by robjtede** [comment, marked as answer]
The macros don't support this currently. Use the manual routing methods.

```rust
.service(web::resource("/hello").get(health))
.service(web::resource("/health").get(health))
```

**Reply by godlockin** [reply]
cheers
